exporter/alertmanagerexporter/README.md

```diff
@@ -0,0 +1,49 @@
+# Alertmanager Exporter
+
+Exports Span Events as alerts to [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/) back-end.
```

Comment:
I feel like there could be an explanation on why specifically SpanEvents, and not Spans themselves.